mindsdb/integrations/handlers/twelve_labs_handler/settings.py

```diff
@@ -182,7 +226,7 @@ class TwelveLabsHandlerConfig(BaseSettings):
         Default wait duration when polling video indexing tasks created via the Twelve Labs API.
     """
 
-    BASE_URL = "https://api.twelvelabs.io/v1.1"
+    BASE_URL = "https://api.twelvelabs.io/v1.2"
```

Comment:
There are some scenarions where users need control over this. Maybe we set it up as optional parameter and this will be default value

Reply:
Done.